components/layouts/DocsLayout.js

```diff
@@ -0,0 +1,7 @@
+export default function DocsLayout({ children }) {
+  return (
+    <div className="float-none mx-auto my-0 w-[90%] max-w-[1440px]">
+      <div className="prose max-w-none w-full mt-8">{children}</div>
+    </div>
```

Comment:
I will try to move the Navbar/footer to here, then we can customise for the docs

Reply:
I think I need to remove a default layout and each "standard" page might need to then select a layout because it seems the `mdx` still uses a default layout and adds another layout on top